### PR TITLE
Update Plex for config flow

### PR DESCRIPTION
**Pull request by jjlawren**
**Description:**
Updated to reflect new use of config flow and removal of deprecated `configurator` setup method.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26548

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/plex.markdown [hunk 1]
```diff
@@ -14,7 +14,7 @@ redirect_from:
 ---
 
 
-The `plex` platform allows you to connect to a [Plex Media Server](https://plex.tv). Once connected, [Plex Clients](https://www.plex.tv/apps-devices/) playing media from the connected Plex Media Server will show up as [Media Players](/components/media_player/) and report playback status via a [Sensor](/components/sensor/) in Home Assistant. The Media Players will allow you to control media playback and see the current playing item.
+The `plex` component allows you to connect to a [Plex Media Server](https://plex.tv). Once connected, [Plex Clients](https://www.plex.tv/apps-devices/) playing media from the connected Plex Media Server will show up as [Media Players](/components/media_player/) and report playback status via a [Sensor](/components/sensor/) in Home Assistant. The Media Players will allow you to control media playback and see the current playing item.
```
Comment (MartinHjelmare) [inline source/_components/plex.markdown]: Maybe call it "integration" instead of component?

Comment (jjlawren) [reply source/_components/plex.markdown]: Sounds good. Changed the instances where "component" was used in a descriptive way.

**Review by MartinHjelmare — DISMISSED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
Please add `ha_config_flow: true` to the frontmatter.

**Review by frenck — APPROVED** [review]
Thanks, @jjlawren! 👍

**Pull request merged**